wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -336,7 +347,9 @@ func (s *WSBroadcastServer) StopAndWait() {
 		log.Warn("error in poller.Stop", "err", err)
 	}
 
+	s.acceptDescMutex.Lock()
 	err = s.acceptDesc.Close()
```

Comment:
I think we need to set this to `nil` after calling close

Reply:
:+1: